Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.12
+
+# Get git
+RUN apt-get update \
+    && apt-get -y install curl git \
+    && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Where pegnet sources will live
+WORKDIR $GOPATH/src/github.com/pegnet/pegnet
+
+# Get goveralls for testing/coverage
+RUN go get github.com/mattn/goveralls
+
+# Populate the rest of the source
+COPY . .
+
+ARG GOOS=linux
+ENV GO111MODULE=on
+
+# Setup the cache directory
+RUN mkdir -p /root/.pegnet/
+COPY defaultconfig.ini /root/.pegnet/defaultconfig.ini
+
+RUN go run Initialization/InitalizeTheNetwork.go
```

Comment:
will this work as at this point no EC are available?

Reply:
This is not 100% necessary and will probably change in the future, this was my way of getting some stuff installed in the image.